src/Controls/src/Core/Platform/Android/PointerGestureHandler.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using Android.Views;
+using Microsoft.Maui.Controls.Platform.Android;
+using Microsoft.Maui.Graphics;
+using AView = Android.Views.View;
+
+namespace Microsoft.Maui.Controls.Platform
+{
+	internal class PointerGestureHandler
+	{
+		internal PointerGestureHandler(Func<View> getView, Func<AView> getControl)
+		{
+			GetView = getView;
+			GetControl = getControl;
+			SetupHandlerForPointer();
+		}
+
+		Func<View> GetView { get; }
+		Func<AView> GetControl { get; }
+
+		public void SetupHandlerForPointer()
+		{
+			var view = GetView();
+			if (view == null)
+				return;
+
+			var control = GetControl();
+			if (control == null)
+				return;
+
+			foreach (var gesture in view.GetCompositeGestureRecognizers())
+				if (gesture is PointerGestureRecognizer)
+					OnPointerAction(view, control, gesture as PointerGestureRecognizer);
```

Comment:
1) This code doesn't need to get called for each `PointerGestureRecognizer` that's  found it only needs to be called if one PointerGesture exists. All that matters is that there's one which means you need to enable the pointer tracking
2) if no pointer gestures are found then I think it makes sense to clear the `SetOnHoverListener`